features/step_definitions/project_steps.rb

```diff
@@ -43,12 +43,14 @@
   end
 end
 
-When("the project is approved") do
-  app.project_under_test.approve
-end
-
-When("the project is rejected") do
-  app.project_under_test.reject
+When(/the project is (approved|rejected)/) do |action|
```

Comment:
Would changing this to
`When(/the project receives (approval|rejection)/) do |factory_name|`
allow you to skip the case and jump straight to `build(factor_name.to_sym)`?

Reply:
yep!